lib/teiserver/battle/libs/balance_lib.ex

```diff
@@ -704,7 +705,27 @@ defmodule Teiserver.Battle.BalanceLib do
 
   @spec calculate_rating_value(float(), float()) :: float()
   def calculate_rating_value(skill, uncertainty) do
-    max(skill - uncertainty, 0)
+    calculate_rating_value(skill, uncertainty, 0)
+  end
+
+  @spec calculate_rating_value(float(), float(), integer()) :: float()
+  def calculate_rating_value(skill, uncertainty, num_matches) do
+    if new_players_start_at_zero?() do
+      num_matches_target = get_num_matches_for_rating_to_equal_skill()
+
+      min(1, num_matches / num_matches_target) * skill
+    else
+      max(skill - uncertainty, 0)
+    end
+  end
+
+  def new_players_start_at_zero?() do
+    config = Config.get_site_config_cache("hidden.Rating method")
+    config == "start at zero; converge to skill"
+  end
+
+  def get_num_matches_for_rating_to_equal_skill() do
+    Config.get_site_config_cache("profile.Num matches for rating to equal skill")
   end
```

Comment:
Not sure about functions that just call another function, I guess it could make sense if there is some additional logic or some other considerations but I don't think it's true in this case. What do you think?